src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.scss

```diff
@@ -1,6 +1 @@
 @import '../../shared/exercise/_exercise-update.scss';
-
```

Comment:
I think you can remove this file and use the _exercise-update.scss instead

Reply:
good idea. I removed all of them. @julian-christl